bundles/org.openhab.binding.clementineremote/NOTICE

```diff
@@ -0,0 +1,13 @@
+This content is produced and maintained by the openHAB project.
+
+* Project home: https://www.openhab.org
+
+== Declared Project Licenses
```

Comment:
No license expert here, just want to mention that i see a different license for the auto generated files like: [/src/main/java/de/qspool/clementineremote/backend/pb/ClementineRemote.java](https://github.com/openhab/openhab-addons/pull/14110/files#diff-0a7b44b39207504251b525e44640229058e08e3dd18dd98b89d1c81dc28e13cb)

The dependency on com.google.protobuf might need to be mentioned, but as said, i'm no expert here.

Reply:
I've updated the NOTICE file in a recent commit. But I'm no expert in these things. Again, I would be for any suggestion how to improve this text.

Reply:
@jlaur can you advice?